### DISCUSSION TITLE
How to set max Upload Size

### DISCUSSION THREAD

**Discussion by gomaaz** [ZoeyVid/NPMplus#1165] [Q&A] [answered]
Hi, 

I wonder how to set the max upload size to my different proxy hosts.
Is there a value by default already set?

**Comment by gomaaz** [comment]
I noticed, Modsecurity = on, causing a limit of about 12 MB...

**Reply by Zoey2936** [reply]
You can edit this in the modsec conf or disable it for the host

**Comment by Zoey2936** [comment, marked as answer]
https://github.com/ZoeyVid/NPMplus/blob/1bd7705ca315b78619ee279a30b296d7d1443d6a/rootfs/usr/local/nginx/conf/nginx.conf#L37 the default is unlimited (0 means unlimited), you can change this value per host using the advanced config

**Comment by gomaaz** [comment]
Thanks!
